src/analysis/record.rs

```diff
@@ -77,11 +78,12 @@ pub fn new(env: &Env, obj: &GObject) -> Option<Info> {
     let mut imports = Imports::with_defined(&env.library, &name);
     imports.add("glib::translate::*", None);
     imports.add("ffi", None);
-    imports.add("glib_ffi", None);
-    imports.add("std::mem", None);
-    imports.add("std::ptr", None);
     if record.glib_get_type.is_some() {
-        imports.add("gobject_ffi", None);
+        if let RecordType::AutoBoxed = RecordType::of(&record) {
+            imports.add("gobject_ffi", None);
+        } else if use_boxed_functions {
+            imports.add("gobject_ffi", None);
```

Comment:
Not better do fast check "use_boxed_function" first and then do slow "RecordTypeOf" ?

Reply:
Done